site.conf

```diff
@@ -15,8 +15,8 @@
 # Pd, iOS, OpenFrameworks, etc.  See "README.embed" for more information.
 #BUILDTYPE = STANDALONE
 #BUILDTYPE = OSXEMBEDDED
-BUILDTYPE = MAXMSP
-#BUILDTYPE = PD
+#BUILDTYPE = MAXMSP
+BUILDTYPE = PD
```

Comment:
Yes, I think Brad had switched it.  If you take out your change to that setting, I will switch it back to STANDALONE with a subsequent change to master.